src/lib/logger.js

```diff
@@ -0,0 +1,8 @@
+import minilog from 'minilog'
+
+const logger = minilog('cozy-drive')
```

Comment:
Since this logger is for cozy-drive (and not photo) maybe I should move it to drive/lib ?

Reply:
Well it will be used on photos right away, because of the sharing module for example. Maybe we can define a new webpack variable for this?